multimodal/src/autogluon/multimodal/learners/base.py

```diff
@@ -1189,9 +1193,9 @@ def run_trainer(
                 return outputs
 
     def on_fit_per_run_start(self, seed, save_path):
-        pl.seed_everything(seed, workers=True)
         # TODO(?) We should have a separate "_pre_training_event()" for logging messages.
-        logger.info(get_fit_start_message(save_path, self._validation_metric_name))
+        logger.info(on_per_fit_run_start_message(save_path, self._validation_metric_name))
```

Comment:
Do we want to make the name consistent? i.e., `on_fit_per_run_start_message`

Reply:
Good call! Updated.